src/welcome/assets/components/TourPage.tsx

```diff
@@ -17,35 +17,48 @@ export default class TourPage extends Component<{
         this.state = {
             step: 0
         };
+
+        this.steps = [
+            this.getStartingPage(),
+            ...this.getContentPages(),
+            this.getEndingPage()
+        ];
+
+        this.timer = undefined;
     }
 
+    private steps: JSX.Element[];
+    // auto navigate to welcome page in last step
+    private timer: number | undefined;
```

Comment:
Shouldn't it declared as `NodeJS.Timer`?

Reply:
`NodeJS.Timer` failed webpack compilation.  In Javascript, `settimeout` returns id of the timer, thus it's a `number`. In NodeJS, it's a `Timer`.  Here this is used in a webview, so `number` type in plain JS should be used.

Reply:
I see. That makes sense.